app/services/gpo_daily_test_sender.rb

```diff
@@ -24,10 +24,7 @@ def otp_from_date(date = Time.zone.today)
 
   # @return [Hash]
   def designated_receiver_pii
-    @designated_receiver_pii ||= JSON.parse(
-      AppConfig.env.gpo_designated_receiver_pii,
-      symbolize_names: true,
-    )
+    @designated_receiver_pii ||= IdentityConfig.store.gpo_designated_receiver_pii.symbolize_keys
```

Comment:
We have an options: `symbolize_names: true` In the config object we can use here instead of `symbolize_keys`

Reply:
Oooh fancy! willfix